integrations/hightouch/manifest.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "Hightouch",
+  "version": "1.0.0",
+  "overview": "Sync data from any source to LaunchDarkly segments.",
+  "description": "Sync Hightouch models to LaunchDarkly segments to seamlessly maintain your LaunchDarkly feature flags.",
+  "author": "Hightouch",
+  "supportEmail": "developers@hightouch.com",
+  "links": {
+    "site": "https://hightouch.com/",
+    "privacyPolicy": "https://hightouch.com/privacy-policy"
```

```suggestion
    "privacyPolicy": "https://hightouch.com/privacy-policy",
    "launchdarklyDocs": "https://docs.launchdarkly.com/integrations/hightouch"
```

If we can get this suggestion commited we're good to merge from our end for your testing. Can you confirm to me via email where the end documentation will be on your site so I can get those docs ready?